google/gax/_grpc_oauth2client.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2015, Google Inc.
```

Comment:
It'll stay around until we formally deprecate oauth2client. I'll also be adding another module named `_grpc_google_auth.py` that'll be a very small interface to `google.auth`.